src/client/shootline.cpp

```diff
@@ -0,0 +1,46 @@
+/*
+Minetest
+Copyright (C) 2023 Josiah VanderZee <josiah_vanderzee@mediacombb.net>
```

Comment:
You forgot DS who originally wrote the code (5f24a3c0c7630b2bad530aab7deeaa9468c59fd9).

Reply:
Oh no, thank you for catching this!